docs/diagrams/src/README.md

```diff
@@ -0,0 +1,12 @@
+## How do we organize and display data on data.gov?
+
+- Navigating to the [datasets](https://catalog.data.gov/dataset) page we see the following "filters"
```

Comment:
I'm cool with "search facets". The only reason I used the term `"filters"` is because that `div` in the UI ( see pic below ) has a class name of `filters`. 

![Screenshot 2023-08-25 at 10 31 33 AM](https://github.com/GSA/datagov-harvesting-logic/assets/92057378/f63c836d-4f3a-4c1c-bc68-3ce46939c90f)